packages/manager/modules/models/service/Service.class.js

```diff
@@ -103,7 +103,7 @@ export default class Service {
   isEngaged() {
     return (
       !!this.billing.engagement ||
-      !!this.billing.pricing.engagementConfiguration
+      !!(this.billing.pricing && this.billing.pricing.engagementConfiguration)
```

Comment:
```suggestion
      !!(this.billing.pricing?.engagementConfiguration)
```

Reply:
You're right! But currently we are facing an issue with Rollup config that's why, as an Hotfix, I've picked this solution.

Reply:
Related to https://github.com/ovh/manager/issues/4099

Reply:
#5064 solves this issue.
Still need some more investigation. We kill keep you posted.
Thank you!